doc/templates/uno/component/navbar.scss

```diff
@@ -119,13 +115,30 @@
         font-size: 19px;
         line-height: 17px;
       }
-    }
 
+      // Media queries for font size and padding adjustments
```

Comment:
@marcoscodnet  Could we align the banner buttons with the Uno Platform website for consistency? This includes matching the style, font, and size adjustments based on the same breakpoints. 
(As we can switch from one site to another and it should match the same visually if possible)

https://github.com/user-attachments/assets/1ce7439a-3460-4b53-b06c-f956f1153c3e